src/batch/src/executor/insert.rs

```diff
@@ -86,11 +86,10 @@ impl InsertExecutor {
             assert!(data_chunk.visibility().is_none());
 
             // add row-id column as first column
-            let row_ids = source_desc.next_row_id_batch(len);
             let mut builder = I64ArrayBuilder::new(len);
-            row_ids
-                .into_iter()
-                .for_each(|row_id| builder.append(Some(row_id)).unwrap());
+            for _ in 0..len {
+                builder.append(None).unwrap();
```

Comment:
So when to insert row_id?

Reply:
That's the simplest way to allocate row id for table source. Otherwise we should introduce some other workaround like add field for `InsertExecutor` and maintain IdGenerators in local source manager.

Reply:
I'm expecting @TennyZhuang 's "op on chunk" design. 🥵